language-plutus-core/src/Language/PlutusCore/Constant/Apply.hs

```diff
@@ -218,6 +220,13 @@ applyTypedBuiltinName
     => TypedBuiltinName a r -> a -> [Value TyName Name ()] -> QuoteT (Evaluate m) ConstAppResult
 applyTypedBuiltinName (TypedBuiltinName _ schema) = applyTypeSchemed schema
 
+-- we default to 'False' in accordance with the spec
```

Comment:
Doesn't say much to me. What does this `False` mean?

Reply:
Also this is a bit far from what it's talking about!

And I think we decided it should be `error`, but I'm not sure how to fit that into the current machinery?

Reply:
> And I think we decided it should be `error`, but I'm not sure how to fit that into the current machinery?

I think I have an idea, but it requires a bit of coding, so I propose we merge this version first and then I'll change it.

Reply:
Will do.